recipes/hap.py/build.sh

```diff
@@ -18,9 +18,10 @@ mkdir -p $SRC_DIR/build/include/htslib
 mkdir -p $SRC_DIR/build/bin
 touch $SRC_DIR/build/bin/samtools
 touch $SRC_DIR/build/bin/bcftools
+touch $SRC_DIR/build/bin/rtg
 
 cmake ../ -DCMAKE_INSTALL_PREFIX=$PREFIX \
   -DBOOST_ROOT="${PREFIX}" -DBoost_NO_SYSTEM_PATHS=ON -DCMAKE_BUILD_TYPE=Release \
-  -DHTSLIB_ROOT="${PREFIX}"
+  -DHTSLIB_ROOT="${PREFIX}" -DBOOST_LIBRARYDIR="${PREFIX}" -DBoost_USE_STATIC_LIBS=OFF
```

Comment:
It will maybe necessary to add the following:
`
BOOST_INC="/usr/include"\
BOOST_LIB_IO="-lboost_iostreams" \
BOOST_LIB_PO="-lboost_program_options" \
BOOST_LIB_SE="-lboost_serialization"
`

Reply:
Unfortunately...